tests/Fixtures/VariableNaming.php

```
@@ -65,6 +71,7 @@ class ClassWithPromotedProperties {
   public function __construct(
     public string $promotedPropertyOne,
     public string $promoted_property_two,
+    public ?\DOMDocument $promotedNullableProperty = NULL,
```

PHPMD (2.15.0)

Avoid excessively long variable names like $promotedNullableProperty. Keep variable name length under 20. (undefined)

(LongVariable)